tiledb/sm/global_state/signal_handlers.cc

```diff
@@ -112,18 +112,17 @@ static BOOL WINAPI win_ctrl_handler(DWORD dwCtrlType) {
   return false;
 }
 
-Status SignalHandlers::initialize() {
+void SignalHandlers::initialize() {
   if (signal(SIGINT, tiledb_signal_handler) == SIG_ERR) {
-    return Status_Error(
+    throw std::logic_error(
```

Comment:
This should be `runtime_error`. It's about an external OS operation, not anything we're detecting that's internally inconsistent.